### ISSUE TITLE
Effective scheduling

### ISSUE THREAD

**Issue by alessio-balda** [Luigi1992/save-me#25]
The scheduling of the events it's not actually effective. The study sessions must be inserted in a smart way.

**Comment by alicjab** [comment]
The first step is creating only 5, 10, 15 studying sessions before exam (for easy, medium, hard). When you add the exam a month before the date no point of creating 30 studying sessions.

**Comment by alessio-balda** [comment]
It'd be the best if users can decide the starting date and the ending date for creating study sessions

**Comment by alicjab** [comment]
No, it complicates the ui and the app. They want to study before right before the exam, right?

**Comment by alicjab** [comment]
I'll add settings where they can specify the number of days they need to prepare to exam, default now is 5, 10, 15. It's the beginning of scheduling, later, if before the exam they already have study sessions, I will add them earlier

**Comment by alessio-balda** [comment]
Students don't always study right before the exam... For example if one has a trip the week before the exam, he will study the week before he leaves. It shouldn't be too hard to do.

However this is how I think it should be in the end. For now, it's okay to try an easy way to do this.

**Comment by alicjab** [comment]
If the student has this trip in a calendar, the app will know that he's busy and will schedule studying before the trip.

**Comment by Luigi1992** [comment]
Sorry Alicja, but we worked out three months for putting 5, 10 or 15 study sessions on calendar? I think the user can simply go on Google Calendar online and put manually the events. I think he saves time doing this way...

**Comment by alicjab** [comment]
You are right.
